auto_submit/test/requests/github_webhook_test.dart

```diff
@@ -22,7 +23,7 @@ void main() {
             'header1': 'header value1',
           },
           body: webhookEventMock);
-      githubWebhook = GithubWebhook();
+      githubWebhook = GithubWebhook(Config());
```

Comment:
Not needed in this PR, but you may need to create a fake config to inject mocked services for future tests.

Reply:
Hi Keyong, yes, I already wrote a fake config for the future pr. Here I just keep it like this, and I will modify it to the fake config immediately in the following pr.